### PR TITLE
Update tutorial with notes about better mobile support

### PR DESCRIPTION
**Pull request by Jacalz**
This now has the necessary code to also open the numerical keyboard on mobile. I think this tutoial is shaping up very well and is starting to be very useful for many reasons :)

### CODE REVIEW

File: tutorial/numerical-entry.md [hunk 1]
```diff
@@ -72,6 +72,14 @@ func (e *numericalEntry) TypedShortcut(shortcut fyne.Shortcut) {
 }
 ```
 
+As a bonus, we can also make sure that mobile operating systems open the numerical keyboard instead of the default keyboard. This can be done by by first importng the `fyne.io/fyne/v2/driver/mobile` package and overwriting the `Keyboard() mobile.KeyboardType` method that is part of the `m̀obile.Keyboardable` interface. Inside the function, we then simply return the `m̀obile.NumberKeyboard` type.
```
Comment (andydotxyz) [inline tutorial/numerical-entry.md:75]: A typo on m̀ I think?

File: tutorial/numerical-entry.md [hunk 2]
```diff
@@ -124,4 +137,5 @@ func main() {
 	w.SetContent(entry)
 	w.ShowAndRun()
 }
+
```
Comment (andydotxyz) [inline tutorial/numerical-entry.md:140]: Can probably lose this newline addition.

**Review by andydotxyz — CHANGES_REQUESTED** [review]
Nice addition thanks, just an accidental keystroke on an `m` I think.

File: tutorial/numerical-entry.md (hunk 1 above)
Comment (Jacalz) [reply tutorial/numerical-entry.md]: Oh yes, certainly. Nice catch.

File: tutorial/numerical-entry.md (hunk 2 above)
Comment (Jacalz) [reply tutorial/numerical-entry.md]: Yup. Removed now.

**Review by andydotxyz — APPROVED** [review]
Nice thanks

**Pull request merged**